dao/src/test/java/com/fitgoal/dao/impl/UserDaoImplTest.java

```diff
@@ -0,0 +1,120 @@
+package com.fitgoal.dao.impl;
+
+import com.fitgoal.dao.UserDao;
+import com.fitgoal.dao.domain.UserDto;
+import com.fitgoal.dao.util.MySqlTestHelper;
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+import org.apache.ibatis.session.SqlSessionManager;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.testcontainers.containers.MySQLContainer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Optional;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserDaoImplTest {
+
+    @ClassRule
+    public static MySQLContainer mySQLContainer = (MySQLContainer) new MySQLContainer("mysql:latest")
+            .withDatabaseName("test_db")
+            .withInitScript("sqlScripts/create-db-schema.sql");
+
+    private static UserDao dao;
+    private static MySqlTestHelper testHelper;
+
+    @BeforeClass
+    public static void setUpSessionManager() {
```

Comment:
Prepare for a naming convention shitstorm: you do much more than `setUpSessionManager`, so you can use more general `setUp` because you prepare the prerequisites for the test. The content of the method doesn't represent actual work that it does.

Reply:
Renamed